lte/gateway/deploy/agw_install.sh

```diff
@@ -36,7 +36,8 @@ if [[ ! $INTERFACES == *'eth0'*  ]] || [[ ! $INTERFACES == *'eth1'* ]] || ! grep
   sed -i 's/GRUB_CMDLINE_LINUX=""/GRUB_CMDLINE_LINUX="net.ifnames=0 biosdevname=0"/g' /etc/default/grub
   # changing interface name
   grub-mkconfig -o /boot/grub/grub.cfg
-  sed -i 's/enp1s0/eth0/g' /etc/network/interfaces
```

Comment:
should we keep this around and just make sure that it never fails? `sed || true` for example. or does the `auto eth0` handle this

Reply:
yes auto should handle this. 
The sed || true is a nice option but that one gives us more flexibility.

Reply:
auto will fix it. 
`Sed || true` is a good option but that one offered by @shanku9  gives us more flexibility in terms of conf